maldoca/service/maldoca_client_cli.cc

```diff
@@ -37,7 +37,7 @@ ABSL_FLAG(uint16_t, port, 6666, "port for communication");
 ABSL_FLAG(std::string, host, "localhost", "Host address");
 ABSL_FLAG(std::string, input_file, "", "input file");
 #ifndef MALDOCA_CHROME
-ABSL_FLAG(std::string, output_file, "", "output file, if empty goes to stdout");
+ABSL_FLAG(std::string, output_file, "", "output file, if empty goes to stdout ");
```

Comment:
remove extra space at the end.

Reply:
was just a test to trigger the pre-submit. idk y github still shows the outdated file...? anyways, already reverted :)